.github/workflows/main.yml

```diff
@@ -72,6 +71,77 @@ jobs:
         run: |
           . .venv/bin/activate
           pip uninstall -y ai2-catwalk
+  tests:
+    name: Test - suite ${{ matrix.test_suite.name }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        test_suite:
+          - name: A
+            mark: "not suite_B and not suite_C and not suite_D"
+
+          - name: B
+            mark: "suite_B"
+
+          - name: C
+            mark: "suite_C"
+
+          - name: D
+            mark: "suite_D"
```

Comment:
What is this for? More parallelism?

Reply:
Yea, as I mentioned in the description I broke the tests up into arbitrary groups aka "suites".